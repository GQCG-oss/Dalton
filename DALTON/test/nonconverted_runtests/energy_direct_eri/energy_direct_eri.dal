**DALTON INPUT
.RUn wAVE FUNCTION
.DiRECT
.RUNERI
**WAVE FUNCTIONS
.HF
**end of input
