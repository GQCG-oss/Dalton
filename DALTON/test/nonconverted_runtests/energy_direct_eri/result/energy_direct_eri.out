

     ************************************************************************
     *************** Dalton - An Electronic Structure Program ***************
     ************************************************************************

    This is output from DALTON (Release Dalton2013 patch 0)
   ----------------------------------------------------------------------------
    NOTE:
     
    Dalton is an experimental code for the evaluation of molecular
    properties using (MC)SCF, DFT, CI, and CC wave functions.
    The authors accept no responsibility for the performance of
    the code or for the correctness of the results.
     
    The code (in whole or part) is provided under a licence and
    is not to be reproduced for further distribution without
    the written permission of the authors or their representatives.
     
    See the home page "http://daltonprogram.org" for further information.
     
    If results obtained with this code are published,
    the appropriate citations would be both of:
     
       K. Aidas, C. Angeli, K. L. Bak, V. Bakken, R. Bast,
       L. Boman, O. Christiansen, R. Cimiraglia, S. Coriani,
       P. Dahle, E. K. Dalskov, U. Ekstroem, T. Enevoldsen,
       J. J. Eriksen, P. Ettenhuber, B. Fernandez, L. Ferrighi,
       H. Fliegl, L. Frediani, K. Hald, A. Halkier, C. Haettig,
       H. Heiberg, T. Helgaker, A. C. Hennum, H. Hettema,
       E. Hjertenaes, S. Hoest, I.-M. Hoeyvik, M. F. Iozzi,
       B. Jansik, H. J. Aa. Jensen, D. Jonsson, P. Joergensen,
       J. Kauczor, S. Kirpekar, T. Kjaergaard, W. Klopper,
       S. Knecht, R. Kobayashi, H. Koch, J. Kongsted, A. Krapp,
       K. Kristensen, A. Ligabue, O. B. Lutnaes, J. I. Melo,
       K. V. Mikkelsen, R. H. Myhre, C. Neiss, C. B. Nielsen,
       P. Norman, J. Olsen, J. M. H. Olsen, A. Osted,
       M. J. Packer, F. Pawlowski, T. B. Pedersen, P. F. Provasi,
       S. Reine, Z. Rinkevicius, T. A. Ruden, K. Ruud, V. Rybkin,
       P. Salek, C. C. M. Samson, A. Sanchez de Meras, T. Saue,
       S. P. A. Sauer, B. Schimmelpfennig, K. Sneskov,
       A. H. Steindal, K. O. Sylvester-Hvid, P. R. Taylor,
       A. M. Teale, E. I. Tellgren, D. P. Tew, A. J. Thorvaldsen,
       L. Thoegersen, O. Vahtras, M. A. Watson, D. J. D. Wilson,
       M. Ziolkowski and H. Agren.
       The Dalton quantum chemistry program system.
       WIREs Comput. Mol. Sci. 2013. doi: 10.1002/wcms.1172
    
    and
    
       Dalton, a Molecular Electronic Structure Program,
       Release DALTON2013.0 (2013), see http://daltonprogram.org
   ----------------------------------------------------------------------------

    Authors in alphabetical order (major contribution(s) in parenthesis):

  Kestutis Aidas,           Vilnius University,           Lithuania   (QM/MM)
  Celestino Angeli,         University of Ferrara,        Italy       (NEVPT2)
  Keld L. Bak,              UNI-C,                        Denmark     (AOSOPPA, non-adiabatic coupling, magnetic properties)
  Vebjoern Bakken,          University of Oslo,           Norway      (DALTON; geometry optimizer, symmetry detection)
  Radovan Bast,             KTH Stockholm                 Sweden      (DALTON installation and execution frameworks)
  Linus Boman,              NTNU,                         Norway      (Cholesky decomposition and subsystems)
  Ove Christiansen,         Aarhus University,            Denmark     (CC module)
  Renzo Cimiraglia,         University of Ferrara,        Italy       (NEVPT2)
  Sonia Coriani,            University of Trieste,        Italy       (CC module, MCD in RESPONS)
  Paal Dahle,               University of Oslo,           Norway      (Parallelization)
  Erik K. Dalskov,          UNI-C,                        Denmark     (SOPPA)
  Thomas Enevoldsen,        Univ. of Southern Denmark,    Denmark     (SOPPA)
  Janus J. Eriksen,         Aarhus University,            Denmark     (PE-MP2/SOPPA, TDA)
  Berta Fernandez,          U. of Santiago de Compostela, Spain       (doublet spin, ESR in RESPONS)
  Lara Ferrighi,            Aarhus University,            Denmark     (PCM Cubic response)
  Heike Fliegl,             University of Oslo,           Norway      (CCSD(R12))
  Luca Frediani,            UiT The Arctic U. of Norway,  Norway      (PCM)
  Bin Gao,                  UiT The Arctic U. of Norway,  Norway      (Gen1Int library)
  Christof Haettig,         Ruhr-University Bochum,       Germany     (CC module)
  Kasper Hald,              Aarhus University,            Denmark     (CC module)
  Asger Halkier,            Aarhus University,            Denmark     (CC module)
  Hanne Heiberg,            University of Oslo,           Norway      (geometry analysis, selected one-electron integrals)
  Trygve Helgaker,          University of Oslo,           Norway      (DALTON; ABACUS, ERI, DFT modules, London, and much more)
  Alf Christian Hennum,     University of Oslo,           Norway      (Parity violation)
  Hinne Hettema,            University of Auckland,       New Zealand (quadratic response in RESPONS; SIRIUS supersymmetry)
  Eirik Hjertenaes,         NTNU,                         Norway      (Cholesky decomposition)
  Maria Francesca Iozzi,    University of Oslo,           Norway      (RPA)
  Brano Jansik              Technical Univ. of Ostrava    Czech Rep.  (DFT cubic response)
  Hans Joergen Aa. Jensen,  Univ. of Southern Denmark,    Denmark     (DALTON; SIRIUS, RESPONS, ABACUS modules, London, and much more)
  Dan Jonsson,              UiT The Arctic U. of Norway,  Norway      (cubic response in RESPONS module)
  Poul Joergensen,          Aarhus University,            Denmark     (RESPONS, ABACUS, and CC modules)
  Joanna Kauczor,           Linkoeping University,        Sweden      (Complex polarization propagator (CPP) module)
  Sheela Kirpekar,          Univ. of Southern Denmark,    Denmark     (Mass-velocity & Darwin integrals)
  Wim Klopper,              KIT Karlsruhe,                Germany     (R12 code in CC, SIRIUS, and ABACUS modules)
  Stefan Knecht,            ETH Zurich,                   Switzerland (Parallel CI and MCSCF)
  Rika Kobayashi,           Australian National Univ.,    Australia   (DIIS in CC, London in MCSCF)
  Henrik Koch,              NTNU,                         Norway      (CC module, Cholesky decomposition)
  Jacob Kongsted,           Univ. of Southern Denmark,    Denmark     (Polarizable embedding, QM/MM)
  Andrea Ligabue,           University of Modena,         Italy       (CTOCD, AOSOPPA)
  Ola B. Lutnaes,           University of Oslo,           Norway      (DFT Hessian)
  Juan I. Melo,             University of Buenos Aires,   Argentina   (LRESC, Relativistic Effects on NMR Shieldings)
  Kurt V. Mikkelsen,        University of Copenhagen,     Denmark     (MC-SCRF and QM/MM)
  Rolf H. Myhre,            NTNU,                         Norway      (Cholesky, subsystems and ECC2)
  Christian Neiss,          Univ. Erlangen-Nuernberg,     Germany     (CCSD(R12))
  Christian B. Nielsen,     University of Copenhagen,     Denmark     (QM/MM)
  Patrick Norman,           Linkoeping University,        Sweden      (Cubic response and complex response in RESPONS)
  Jeppe Olsen,              Aarhus University,            Denmark     (SIRIUS CI/density modules)
  Jogvan Magnus H. Olsen,   Univ. of Southern Denmark,    Denmark     (Polarizable embedding, PE library, QM/MM)
  Anders Osted,             Copenhagen University,        Denmark     (QM/MM)
  Martin J. Packer,         University of Sheffield,      UK          (SOPPA)
  Filip Pawlowski,          Kazimierz Wielki University   Poland      (CC3)
  Thomas B. Pedersen,       University of Oslo,           Norway      (Cholesky decomposition)
  Patricio F. Provasi,      University of Northeastern,   Argentina   (Analysis of coupling constants in localized orbitals)
  Zilvinas Rinkevicius,     KTH Stockholm,                Sweden      (open-shell DFT, ESR)
  Elias Rudberg,            KTH Stockholm,                Sweden      (DFT grid and basis info)
  Torgeir A. Ruden,         University of Oslo,           Norway      (Numerical derivatives in ABACUS)
  Kenneth Ruud,             UiT The Arctic U. of Norway,  Norway      (DALTON; ABACUS magnetic properties and  much more)
  Pawel Salek,              KTH Stockholm,                Sweden      (DALTON; DFT code)
  Claire C. M. Samson       University of Karlsruhe       Germany     (Boys localization, r12 integrals in ERI)
  Alfredo Sanchez de Meras, University of Valencia,       Spain       (CC module, Cholesky decomposition)
  Trond Saue,               Paul Sabatier University,     France      (direct Fock matrix construction)
  Stephan P. A. Sauer,      University of Copenhagen,     Denmark     (SOPPA(CCSD), SOPPA prop., AOSOPPA, vibrational g-factors)
  Bernd Schimmelpfennig,    Forschungszentrum Karlsruhe,  Germany     (AMFI module)
  Kristian Sneskov,         Aarhus University,            Denmark     (QM/MM, PE-CC)
  Arnfinn H. Steindal,      UiT The Arctic U. of Norway,  Norway      (parallel QM/MM)
  K. O. Sylvester-Hvid,     University of Copenhagen,     Denmark     (MC-SCRF)
  Peter R. Taylor,          VLSCI/Univ. of Melbourne,     Australia   (Symmetry handling ABACUS, integral transformation)
  Andrew M. Teale,          University of Nottingham,     England     (DFT-AC, DFT-D)
  David P. Tew,             University of Bristol,        England     (CCSD(R12))
  Olav Vahtras,             KTH Stockholm,                Sweden      (triplet response, spin-orbit, ESR, TDDFT, open-shell DFT)
  David J. Wilson,          La Trobe University,          Australia   (DFT Hessian and DFT magnetizabilities)
  Hans Agren,               KTH Stockholm,                Sweden      (SIRIUS module, RESPONS, MC-SCRF solvation model)
 --------------------------------------------------------------------------------

     Date and time (Linux)  : Sun Sep  8 20:38:16 2013
     Host name              : lpqlx131.ups-tlse.fr                    

 * Work memory size             :    64000000 =  488.28 megabytes.

 * Directories for basis set searches:
   1) /home/bast/DALTON-2013.0-Source/build/test_energy_direct_eri
   2) /home/bast/DALTON-2013.0-Source/build/basis


Compilation information
-----------------------

 Who compiled             | bast
 Host                     | lpqlx131.ups-tlse.fr
 System                   | Linux-3.8.5-201.fc18.x86_64
 CMake generator          | Unix Makefiles
 Processor                | x86_64
 64-bit integers          | OFF
 MPI                      | OFF
 Fortran compiler         | /usr/bin/gfortran
 Fortran compiler version | GNU Fortran (GCC) 4.7.2 20121109 (Red Hat 4.7.2-8)
 C compiler               | /usr/bin/gcc
 C compiler version       | gcc (GCC) 4.7.2 20121109 (Red Hat 4.7.2-8)
 C++ compiler             | /usr/bin/g++
 C++ compiler version     | g++ (GCC) 4.7.2 20121109 (Red Hat 4.7.2-8)
 Static linking           | OFF
 Last Git revision        | f34203295a86316e27f9e7b44f9b6769c4a046c0
 Configuration time       | 2013-09-08 20:31:27.952056


   Content of the .dal input file
 ----------------------------------

**DALTON INPUT                                    
.RUn wAVE FUNCTION                                
.DiRECT                                           
.RUNERI                                           
**WAVE FUNCTIONS                                  
.HF                                               
**end of input                                    


   Content of the .mol file
 ----------------------------

basis                                                                          
cc-pVDZ                                                                        
Direct SCF calculation with/o symmetry                                         
using ERI integral module                                                      
    Atomtypes=2                                                                
        8.    1                                                                
O     0.0 -0.2249058930 0.0                                                    
        1.    2                                                                
H     1.45235 0.899623 0.0                                                     
H    -1.45235 0.899623 0.0                                                     


       *******************************************************************
       *********** Output from DALTON general input processing ***********
       *******************************************************************

 --------------------------------------------------------------------------------
   Overall default print level:    0
   Print level for DALTON.STAT:    1

    AO-direct calculation (in sections where implemented)
    HERMIT 1- and 2-electron integral sections will be executed
    2-elctron integrals are calculated using ERI instead of HERMIT (where possible)
    "Old" integral transformation used (limited to max 255 basis functions)
    Wave function sections will be executed (SIRIUS module)
 --------------------------------------------------------------------------------


   ****************************************************************************
   *************** Output of molecule and basis set information ***************
   ****************************************************************************


    The two title cards from your ".mol" input:
    ------------------------------------------------------------------------
 1: Direct SCF calculation with/o symmetry                                  
 2: using ERI integral module                                               
    ------------------------------------------------------------------------

  Atomic type no.    1
  --------------------
  Nuclear charge:   8.00000
  Number of symmetry independent centers:    1
  Number of basis sets to read;    2
  Basis set file used for this atomic type with Z =   8 :
     "/home/bast/DALTON-2013.0-Source/build/basis/cc-pVDZ"

  Atomic type no.    2
  --------------------
  Nuclear charge:   1.00000
  Number of symmetry independent centers:    2
  Number of basis sets to read;    2
  Basis set file used for this atomic type with Z =   1 :
     "/home/bast/DALTON-2013.0-Source/build/basis/cc-pVDZ"


                      SYMADD: Requested addition of symmetry
                      --------------------------------------

 Symmetry test threshold:  5.00E-06

 - molecule centered at center of mass and rotated so
   principal axes of inertia are along coordinate axes.

 Symmetry class found: C(2v)          

 Symmetry Independent Centres             
 ----------------------------
       8 :      0.00000000     0.00000000    -0.12585150  Isotope  1
       1 :      0.00000000     1.45235000     0.99867739  Isotope  1

 The following elements were found:   X  Y     


                         SYMGRP: Point group information
                         -------------------------------

Full point group is: C(2v)          
Represented as:      C2v

   * The point group was generated by:

      Reflection in the yz-plane
      Reflection in the xz-plane

   * Group multiplication table

        |  E   C2z  Oxz  Oyz
   -----+--------------------
     E  |  E   C2z  Oxz  Oyz
    C2z | C2z   E   Oyz  Oxz
    Oxz | Oxz  Oyz   E   C2z
    Oyz | Oyz  Oxz  C2z   E 

   * Character table

        |  E   C2z  Oxz  Oyz
   -----+--------------------
    A1  |   1    1    1    1
    B1  |   1   -1    1   -1
    B2  |   1   -1   -1    1
    A2  |   1    1   -1   -1

   * Direct product table

        | A1   B1   B2   A2 
   -----+--------------------
    A1  | A1   B1   B2   A2 
    B1  | B1   A1   A2   B2 
    B2  | B2   A2   A1   B1 
    A2  | A2   B2   B1   A1 


                                 Isotopic Masses
                                 ---------------

                           O          15.994915
                           H   _1      1.007825
                           H   _2      1.007825

                       Total mass:    18.010565 amu
                       Natural abundance:  99.730 %

 Center-of-mass coordinates (a.u.):    0.000000    0.000000    0.000000


  Atoms and basis sets
  --------------------

  Number of atom types :    2
  Total number of atoms:    3

  Basis set used is "cc-pVDZ" from the basis set library.

  label    atoms   charge   prim   cont     basis
  ----------------------------------------------------------------------
  O           1    8.0000    26    14      [9s4p1d|3s2p1d]                                    
  H           2    1.0000     7     5      [4s1p|2s1p]                                        
  ----------------------------------------------------------------------
  total:      3   10.0000    40    24
  ----------------------------------------------------------------------
  Spherical harmonic basis used.

  Threshold for neglecting AO integrals:  1.00D-12


  Cartesian Coordinates (a.u.)
  ----------------------------

  Total number of coordinates:    9
  O       :     1  x   0.0000000000    2  y   0.0000000000    3  z  -0.1258515023
  H   / 1 :     4  x   0.0000000000    5  y   1.4523500000    6  z   0.9986773907
  H   / 2 :     7  x   0.0000000000    8  y  -1.4523500000    9  z   0.9986773907


  Symmetry Coordinates
  --------------------

  Number of coordinates in each symmetry:     3    2    3    1

  Symmetry  A1  ( 1)

    1   O     z    3
    2   H     y    [  5  -    8 ]/2
    3   H     z    [  6  +    9 ]/2

  Symmetry  B1  ( 2)

    4   O     x    1
    5   H     x    [  4  +    7 ]/2

  Symmetry  B2  ( 3)

    6   O     y    2
    7   H     y    [  5  +    8 ]/2
    8   H     z    [  6  -    9 ]/2

  Symmetry  A2  ( 4)

    9   H     x    [  4  -    7 ]/2


   Interatomic separations (in Angstrom):
   --------------------------------------

            O           H   _1      H   _2
            ------      ------      ------
 O     :    0.000000
 H   _1:    0.972000    0.000000
 H   _2:    0.972000    1.537101    0.000000


  Max    interatomic separation is    1.5371 Angstrom (    2.9047 Bohr)
  between atoms    3 and    2, "H   _2" and "H   _1".

  Min HX interatomic separation is    0.9720 Angstrom (    1.8368 Bohr)


  Bond distances (Angstrom):
  --------------------------

                  atom 1     atom 2       distance
                  ------     ------       --------
  bond distance:  H   _1     O            0.972000
  bond distance:  H   _2     O            0.972000


  Bond angles (degrees):
  ----------------------

                  atom 1     atom 2     atom 3         angle
                  ------     ------     ------         -----
  bond angle:     H   _1     O          H   _2       104.500




 Principal moments of inertia (u*A**2) and principal axes
 --------------------------------------------------------

   IA       0.633889          0.000000    1.000000    0.000000
   IB       1.190584          0.000000    0.000000    1.000000
   IC       1.824473          1.000000    0.000000    0.000000


 Rotational constants
 --------------------

 The molecule is planar.

               A                   B                   C

         797267.3370         424479.9976         277000.0210 MHz
           26.593976           14.159129            9.239726 cm-1


@  Nuclear repulsion energy :    9.055004525638 Hartree


  Symmetry Orbitals
  -----------------

  Number of orbitals in each symmetry:          11    4    7    2


  Symmetry  A1 ( 1)

    1     O        1s         1
    2     O        1s         2
    3     O        1s         3
    4     O        2pz        6
    5     O        2pz        9
    6     O        3d0       12
    7     O        3d2+      14
    8     H        1s        15 +   16
    9     H        1s        17 +   18
   10     H        2py       21 -   22
   11     H        2pz       23 +   24


  Symmetry  B1 ( 2)

   12     O        2px        4
   13     O        2px        7
   14     O        3d1+      13
   15     H        2px       19 +   20


  Symmetry  B2 ( 3)

   16     O        2py        5
   17     O        2py        8
   18     O        3d1-      11
   19     H        1s        15 -   16
   20     H        1s        17 -   18
   21     H        2py       21 +   22
   22     H        2pz       23 -   24


  Symmetry  A2 ( 4)

   23     O        3d2-      10
   24     H        2px       19 -   20

  Symmetries of electric field:  B1 (2)  B2 (3)  A1 (1)

  Symmetries of magnetic field:  B2 (3)  B1 (2)  A2 (4)


                     .---------------------------------------.
                     | Starting in Integral Section (HERMIT) |
                     `---------------------------------------'



    *************************************************************************
    ****************** Output from HERMIT input processing ******************
    *************************************************************************



     ************************************************************************
     ************************** Output from HERINT **************************
     ************************************************************************

 >>>> Total CPU  time used in HERMIT:   0.01 seconds
 >>>> Total wall time used in HERMIT:   0.01 seconds


                        .----------------------------------.
                        | End of Integral Section (HERMIT) |
                        `----------------------------------'



                   .--------------------------------------------.
                   | Starting in Wave Function Section (SIRIUS) |
                   `--------------------------------------------'


 *** Output from Huckel module :

     Using EWMO model:          F
     Using EHT  model:          T
     Number of Huckel orbitals each symmetry:    4    1    2    0

 Huckel EHT eigenvalues for symmetry :  1
          -20.704225      -1.499390      -0.671093      -0.296848

 Huckel EHT eigenvalues for symmetry :  2
           -0.616200

 Huckel EHT eigenvalues for symmetry :  3
           -0.731017      -0.261127

 **********************************************************************
 *SIRIUS* a direct, restricted step, second order MCSCF program       *
 **********************************************************************

 
     Date and time (Linux)  : Sun Sep  8 20:38:16 2013
     Host name              : lpqlx131.ups-tlse.fr                    

 Title lines from ".mol" input file:
     Direct SCF calculation with/o symmetry                                  
     using ERI integral module                                               

 Print level on unit LUPRI =   2 is   0
 Print level on unit LUW4  =   2 is   5

@    Restricted, closed shell Hartree-Fock calculation.
 Fock matrices are calculated directly without use of integrals on disk.

 Initial molecular orbitals are obtained according to
 ".MOSTART EHT   " input option

     Wave function specification
     ============================
@    For the wave function of type :      >>> HF <<<
@    Number of closed shell electrons          10
@    Number of electrons in active shells       0
@    Total charge of the molecule               0

@    Spin multiplicity and 2 M_S                1         0
     Total number of symmetries                 4
@    Reference state symmetry                   1

     Orbital specifications
     ======================
     Abelian symmetry species          All |    1    2    3    4
                                       --- |  ---  ---  ---  ---
     Total number of orbitals           24 |   11    4    7    2
     Number of basis functions          24 |   11    4    7    2

      ** Automatic occupation of RHF orbitals **

      -- Initial occupation of symmetries is determined from extended Huckel guess.           
      -- Initial occupation of symmetries is :
@    Occupied SCF orbitals               5 |    3    1    1    0

     Maximum number of Fock   iterations      0
     Maximum number of DIIS   iterations     60
     Maximum number of QC-SCF iterations     60
     Threshold for SCF convergence     1.00D-05


 >>>>> DIIS optimization of Hartree-Fock <<<<<

 C1-DIIS algorithm; max error vectors =   10

 Automatic occupation of symmetries with  10 electrons.

 Iter     Total energy    Error norm  Delta(E)    SCF occupation
 -----------------------------------------------------------------------------
   1  Screening settings (-IFTHRS, DIFDEN)   -6    T
@  1  -75.8080240525       2.26D+00  -7.58D+01     3   1   1   0
      Virial theorem: -V/T =      1.999834
@      MULPOP O      -0.43; H   _1  0.22; H   _2  0.22; 
 -----------------------------------------------------------------------------
   2  Screening settings (-IFTHRS, DIFDEN)   -6    T
@  2  -76.0179559355       3.50D-01  -2.10D-01     3   1   1   0
      Virial theorem: -V/T =      2.000516
@      MULPOP O      -0.28; H   _1  0.14; H   _2  0.14; 
 -----------------------------------------------------------------------------
   3  Screening settings (-IFTHRS, DIFDEN)   -6    T
@  3  -76.0250798476       8.42D-02  -7.12D-03     3   1   1   0
      Virial theorem: -V/T =      2.002196
@      MULPOP O      -0.30; H   _1  0.15; H   _2  0.15; 
 -----------------------------------------------------------------------------
   4  Screening settings (-IFTHRS, DIFDEN)  -10    F
@  4  -76.0256316151       2.94D-02  -5.52D-04     3   1   1   0
      Virial theorem: -V/T =      2.000742
@      MULPOP O      -0.32; H   _1  0.16; H   _2  0.16; 
 -----------------------------------------------------------------------------
   5  Screening settings (-IFTHRS, DIFDEN)  -10    T
@  5  -76.0256787149       7.26D-03  -4.71D-05     3   1   1   0
      Virial theorem: -V/T =      2.001331
@      MULPOP O      -0.32; H   _1  0.16; H   _2  0.16; 
 -----------------------------------------------------------------------------
   6  Screening settings (-IFTHRS, DIFDEN)  -10    T
@  6  -76.0256814298       7.07D-04  -2.71D-06     3   1   1   0
      Virial theorem: -V/T =      2.001210
@      MULPOP O      -0.32; H   _1  0.16; H   _2  0.16; 
 -----------------------------------------------------------------------------
   7  Screening settings (-IFTHRS, DIFDEN)  -10    T
@  7  -76.0256814818       1.30D-04  -5.20D-08     3   1   1   0
      Virial theorem: -V/T =      2.001216
@      MULPOP O      -0.32; H   _1  0.16; H   _2  0.16; 
 -----------------------------------------------------------------------------
   8  Screening settings (-IFTHRS, DIFDEN)  -10    T
@  8  -76.0256814839       1.07D-05  -2.11D-09     3   1   1   0
      Virial theorem: -V/T =      2.001217
@      MULPOP O      -0.32; H   _1  0.16; H   _2  0.16; 
 -----------------------------------------------------------------------------
   9  Screening settings (-IFTHRS, DIFDEN)  -10    T
@  9  -76.0256814839       6.63D-07  -1.20D-11     3   1   1   0

@ *** DIIS converged in   9 iterations !
@     Converged SCF energy, gradient:    -76.025681483944    6.63D-07
    - total time used in SIRFCK :              0.00 seconds


 *** SCF orbital energy analysis ***

 Only the five lowest virtual orbital energies printed in each symmetry.

 Number of electrons :   10
 Orbital occupations :    3    1    1    0

 Sym       Hartree-Fock orbital energies

  1    -20.55301999    -1.32934105    -0.56387096     0.18304191     0.84069368
         1.16451078     1.44696810     1.87052764

  2     -0.49212364     1.20008013     1.67196077     3.27277142

  3     -0.69144632     0.25403523     0.77887080     1.25426150     1.91890925
         2.41834248

  4      1.47255805     3.32446046

    E(LUMO) :     0.18304191 au (symmetry 1)
  - E(HOMO) :    -0.49212364 au (symmetry 2)
  ------------------------------------------
    gap     :     0.67516556 au

 >>> Writing SIRIFC interface file <<<

 >>>> CPU and wall time for SCF :       0.111       0.112


                       .-----------------------------------.
                       | >>> Final results from SIRIUS <<< |
                       `-----------------------------------'


@    Spin multiplicity:           1
@    Spatial symmetry:            1
@    Total charge of molecule:    0

@    Final HF energy:             -76.025681483944                 
@    Nuclear repulsion:             9.055004525638
@    Electronic energy:           -85.080686009582

@    Final gradient norm:           0.000000663348

 
     Date and time (Linux)  : Sun Sep  8 20:38:16 2013
     Host name              : lpqlx131.ups-tlse.fr                    

 (Only coefficients >0.0100 are printed.)

 Molecular orbitals for symmetry species  1
 ------------------------------------------

    Orbital         1        2        3        4        5
   1 O   :1s     1.0004  -0.0076   0.0022   0.0519   0.0683
   2 O   :1s     0.0021   0.8737  -0.2932  -0.1433   0.5085
   3 O   :1s    -0.0014  -0.1145  -0.1768  -0.9058  -0.6107
   4 O   :2pz    0.0025   0.1108   0.7927  -0.2855   0.4774
   5 O   :2pz   -0.0017  -0.0619   0.0018  -0.2117  -0.2176
   6 O   :3d0    0.0000   0.0012   0.0177  -0.0075  -0.0004
   7 O   :3d2+  -0.0001  -0.0029  -0.0046   0.0092   0.1048
   8 H   :1s    -0.0003   0.3191   0.3516   0.1058  -1.3631
   9 H   :1s     0.0006  -0.1494  -0.1327   0.7699   1.2409
  10 H   :2py    0.0005  -0.0364  -0.0317  -0.0162  -0.2838
  11 H   :2pz    0.0005  -0.0201   0.0070  -0.0156  -0.0569

 Molecular orbitals for symmetry species  2
 ------------------------------------------

    Orbital         1        2        3
   1 O   :2px    0.9204  -1.4099  -0.0114
   2 O   :2px    0.0724   1.6625   0.6479
   3 O   :3d1+   0.0180  -0.0024  -0.1619
   4 H   :2px    0.0305   0.0229  -0.7640

 Molecular orbitals for symmetry species  3
 ------------------------------------------

    Orbital         1        2        3
   1 O   :2py    0.7111  -0.4190  -0.3966
   2 O   :2py   -0.1012  -0.4775  -0.2515
   3 O   :3d1-   0.0262  -0.0201   0.1103
   4 H   :1s     0.5530   0.0501   1.6037
   5 H   :1s    -0.1845   1.3912  -1.5126
   6 H   :2py   -0.0234  -0.0198   0.0700
   7 H   :2pz   -0.0324  -0.0157   0.1485

 Molecular orbitals for symmetry species  4
 ------------------------------------------

    Orbital         1        2
   1 O   :3d2-  -0.1334   1.0607
   2 H   :2px   -0.6844  -0.3620



 >>>> Total CPU  time used in SIRIUS :      0.12 seconds
 >>>> Total wall time used in SIRIUS :      0.12 seconds

 
     Date and time (Linux)  : Sun Sep  8 20:38:16 2013
     Host name              : lpqlx131.ups-tlse.fr                    


                     .---------------------------------------.
                     | End of Wave Function Section (SIRIUS) |
                     `---------------------------------------'

 >>>> Total CPU  time used in DALTON:   0.13 seconds
 >>>> Total wall time used in DALTON:   0.13 seconds

 
     Date and time (Linux)  : Sun Sep  8 20:38:16 2013
     Host name              : lpqlx131.ups-tlse.fr                    
